plugin/cache/README.md

```diff
@@ -50,6 +51,10 @@ cache [TTL] [ZONES...] {
   **DURATION** defaults to 1m. Prefetching will happen when the TTL drops below **PERCENTAGE**,
   which defaults to `10%`, or latest 1 second before TTL expiration. Values should be in the range `[10%, 90%]`.
   Note the percent sign is mandatory. **PERCENTAGE** is treated as an `int`.
+* `serve_stale`, when serve\_stale is set, cache always will serve an expired entry to a client if there is one
+  available.  When this happens, cache will attempt to refresh the cache entry after sending the expired cache
+  entry to the client. The responses have a TTL of 0. **DURATION** is how far back to consider
+  stale responses as fresh. The default is: 'serve\_stale 0h'.
```

Comment:
`0h` or did you mean `1h` here?

Reply:
I'm removing that. There is no default now and you have to specify the duration.

Reply:
Per setup.go, looks like duration is optional and default is 1h.

Reply:
Yup, it's updated in my last push.